web/src/components/user/UserGeneralTab.vue

```diff
@@ -3,6 +3,16 @@
     <InputField :label="$t('user.settings.general.language')">
       <SelectField v-model="selectedLocale" :options="localeOptions" />
     </InputField>
+    <InputField :label="$t('user.settings.general.theme.theme')">
```

Comment:
Just as I notice it here again, should we start to stop deeply nesting translations to have it easier to re-use them. So in this case simply `theme.theme`